packages/ui/package.json

```diff
@@ -45,8 +45,8 @@
   "dependencies": {
     "csstype": "^3.1.1",
     "lodash": "4.17.21",
-    "style-dictionary": "3.7.0",
-    "tslib": "2.4.0"
+    "style-dictionary": "^3.7.1",
```

Comment:
I think we should pin those dependencies and upgrade manually?

Reply:
Why do we want to ping the version if it doesn't have any breaking changes?

Reply:
I think we encountered this before. Some dependencies do not follow semver strictly and introduce breaking changes for minor bump. So we pin them afterwards

Reply:
2 scenarios:
A. There's a minor version to fix a security issue, but we missed it.
B. There's a minor version update, but broke our code

I'd rather have scenario B unless there's some special cases that we have to add the pkg to `resolution` to ping the version.

It's weird that AppSec didn't complain about the react pkg as you linked above. And Angular is using `^2.0.0` and `^2.3.1`

Reply:
I agree with @zchenwei here because NOT pinning versions has been shown to break customers and we are already monitoring security updates via dependabot.

Reply:
Yea, what I mean is not breaking our code since we have `yarn.lock`. It will break customer apps

Reply:
+1, we deliberately pinned the versions to prevent regressions in direct dependencies and downstream dependencies.

Reply:
Ok. PR updated!